[3.2.0RC1] IllegalArgumentException  in FilesSyncHelper.scheduleFilesSyncIfNeeded

Reported via GPlay:
```
Caused by: java.lang.IllegalArgumentException: 
  at com.google.android.gms.gcm.GcmNetworkManager.zzie (Unknown Source)
  at com.google.android.gms.gcm.GcmNetworkManager.cancelTask (Unknown Source)
  at com.evernote.android.job.gcm.JobProxyGcm.cancel (JobProxyGcm.java:111)
  at com.evernote.android.job.JobManager.cancelInner (JobManager.java:399)
  at com.evernote.android.job.JobManager.cancelAllInner (JobManager.java:422)
  at com.evernote.android.job.JobManager.cancelAllForTag (JobManager.java:393)
  at com.evernote.android.job.JobManager.schedule (JobManager.java:183)
  at com.evernote.android.job.JobRequest.schedule (JobRequest.java:436)
  at com.owncloud.android.utils.FilesSyncHelper.scheduleFilesSyncIfNeeded (FilesSyncHelper.java:296)
  at com.owncloud.android.MainApp.initSyncOperations (MainApp.java:290)
  at com.owncloud.android.MainApp.onCreate (MainApp.java:159)
  at android.app.Instrumentation.callApplicationOnCreate (Instrumentation.java:1013)
  at android.app.ActivityThread.handleBindApplication (ActivityThread.java:4712)
```

Happens to a bunch of Samsung/HTC devices, at least for now all Android 4.4 which most not be the only Android version getting hit but until now the only one where we got reports for.

--- a/src/gplay/AndroidManifest.xml
+++ b/src/gplay/AndroidManifest.xml
@@ -77,6 +77,17 @@
             </intent-filter>
         </service>
 
+        <service
+            android:name="com.evernote.android.job.gcm.PlatformGcmService"
+            android:enabled="true"
+            android:exported="true"
+            android:permission="com.google.android.gms.permission.BIND_NETWORK_TASK_SERVICE"
+            tools:replace="android:enabled">
+            <intent-filter>
+                <action android:name="com.google.android.gms.gcm.ACTION_TASK_READY"/>
+            </intent-filter>
+        </service>
+
     </application>
 
 </manifest>

